File: Hackerrank/birthday_cake_candles.py
#can only blow out tallest candles, how many can she blow out
def candles(ar):
    tallest = 0
    count = 0
    for x in ar:
        print(x)
        if x == tallest:
            print('== tallest')
            count += 1
        elif x > tallest:
            print('> than ')
            tallest = x
            count = 1
    return count



A = [3,2,1,3]
print(candles(A))

File: Hackerrank/birthday_cake_candles.js
function candles(A){
    //sort array
    A.sort()
    console.log(A)
    // reverse iterate
    //count
    count = 1
    pt = 2
    console.log(A[A.length - pt], A[A.length -1])
    while (A[A.length - pt] === A[A.length - 1]){
       console.log('in count', count)
        count += 1
        pt += 1
    }
    console.log({count})
    return count
}

A = [3,2,3,1,3]
console.log(candles(A))